docs/cloud-providers/azure/azure-provider.md

```diff
@@ -41,26 +41,17 @@ You might see an error similar to the following, but that is OK, the permissions
 Operation failed with status: 'Conflict'. Details: 409 Client Error: Conflict for url: https://graph.windows.net/e7985bc4-a3b3-4f37-b9d2-fa256023b1ae/oauth2PermissionGrants?api-version=1.6
 ```
 
-After these steps are completed, you should have the following file on your local filesystem:
-
-* `crossplane-azure-provider-key.json`
-
-## Grant Consent to Application Permissions
+Finally, you need to grant admin permissions on the Azure Active Directory to the service principal because it will need to create other service principals for your `AKSCluster`:
+```bash
+# gra
```

Comment:
what is this `gra` comment supposed to mean? is it truncated for a more useful comment that starts with `grant...`? :)

Reply:
Thanks for catching! Fixed.